### PR TITLE
Fixes #14 and #18

### LINKED ISSUE #14: Dependency Error in graphiql-spring-boot-starter
`Failed to collect dependencies at com.graphql-java:graphiql-spring-boot-starter:jar:3.3.0 -> com.graphql-java:graphiql-spring-boot-autoconfigure:jar:3.3.0 -> org.webjars.npm:react:jar:15.5.4 -> org.webjars.npm:fbjs:jar:0.9.0-alpha.2 -> org.webjars.npm:fbjs-css-vars:jar:[1.0.0,2): No versions available for org.webjars.npm:fbjs-css-vars:jar:[1.0.0,2) within specified range `

### PR DESCRIPTION
**Pull request by oliemansm**
Replaced javascript dependencies by CDN urls instead of the webjars that appeared to be unstable.

### CODE REVIEW

File: graphiql-spring-boot-autoconfigure/build.gradle [hunk 1]
```diff
@@ -17,9 +17,4 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 dependencies{
-    compile "org.webjars.npm:es6-promise:$LIB_WEBJARS_ES6_PROMISE_VER"
```
Comment (apottere) [inline graphiql-spring-boot-autoconfigure/build.gradle]: Would you mind also removing these properties?

Comment (oliemansm) [inline graphiql-spring-boot-autoconfigure/build.gradle]: Sure, they're removed now. Thanks for checking.

**Comment by apottere** [comment]
Awesome, thanks!  I'll release this to central once I take a look at some other bugs tomorrow.

**Comment by oliemansm** [comment]
Great, thanks!

**Comment by apottere** [comment]
This is released in `3.5.0`, syncing to central now.

**Pull request merged**